### PR TITLE
Added validation Altibase

### LINKED ISSUE #9853: [Bug]: altibase handler integration lacks  exception handling
### Short description of current behavior

The current implementation lacks proper exception handling, as the creation query shows as successfully completed even with incorrect credentials.
found about this while testing this issue #9813 

### Video or screenshots

{481ED6F1-A95E-4F40-B1CC-EE51870CE65D}

### Expected behavior

_No response_

### How to reproduce the error

use the creation query without correct credentials

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by parthiv11**
## Description

Please include a summary of the change and the issue it solves. 

Fixes #9853 

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by parthiv11** [comment]
[altibase_bug.webm](https://github.com/user-attachments/assets/34212799-7ff6-464c-b76d-865f3c4521f3)

**Comment by parthiv11** [comment]
According to documentation `jar_location` is correct

File: mindsdb/integrations/handlers/altibase_handler/altibase_handler.py [hunk 1]
```diff
@@ -96,14 +97,14 @@ def connect_with_jdbc(self):
         Returns:
             connection
         """
-        jar_location = self.connection_args.get('jdbcJarLocation')
+        jar_location = self.connection_args.get('jar_location')
```
[the remark this answers is not in the document]
Comment (parthiv11) [reply mindsdb/integrations/handlers/altibase_handler/altibase_handler.py:100]: According to documentation `jar_location` is correct

https://github.com/mindsdb/mindsdb/blob/253ac77591885137c798ba147f55f2a53b228ab0/mindsdb/integrations/handlers/altibase_handler/README.md?plain=1#L20

[the remark this answers is not in the document]
Comment (ZoranPandovski) [reply mindsdb/integrations/handlers/altibase_handler/altibase_handler.py:100]: Check the `connection_args` https://github.com/mindsdb/mindsdb/blob/253ac77591885137c798ba147f55f2a53b228ab0/mindsdb/integrations/handlers/altibase_handler/connection_args.py#L32.

[the remark this answers is not in the document]
Comment (parthiv11) [reply mindsdb/integrations/handlers/altibase_handler/altibase_handler.py:100]: either README or connection_args need to update.

what you say ?
i think staying with kabab_case is good

[the remark this answers is not in the document]
Comment (ZoranPandovski) [reply mindsdb/integrations/handlers/altibase_handler/altibase_handler.py:100]: Let's use kebab_case. Please update everywhere, same for jdbcClass

**Comment by parthiv11** [comment]
done!!

File: mindsdb/integrations/handlers/altibase_handler/altibase_handler.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (parthiv11) [reply mindsdb/integrations/handlers/altibase_handler/altibase_handler.py:100]: done!!

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**